src/products/firefox_desktop/fluent_migrations.md

```diff
@@ -0,0 +1,83 @@
+# Running Fluent migrations
+
+[Fluent migrations](https://firefox-source-docs.mozilla.org/l10n/migrations/index.html) are used to migrate existing translations from legacy formats (DTD, properties) to Fluent, or existing Fluent messages to new messages or different FTL files. This document describes in detail which operations need to be performed in order to run migrations on over 150 Mercurial repositories for Firefox. Several access rights are required:
+* Write access to [l10n-central](https://hg.mozilla.org/l10n-central), i.e. SSH access with `scm_l10n` rights.
```

Comment:
Actually, `active_scm_l10n`. AFAICT, https://people.mozilla.org/e#nav-access-groups is the least horrible URL to share on how to find out if you have that or not.

Reply:
Interesting, is that different from the permissions on the repo? That's `scm_l10n`, or at least that's the one we ask for when requesting new repos.

Didn't even notice those permissions are visible in PMO.

Reply:
`scm_l10n` can be deactivated for the lack of use of hgmo access, which is why the `_active` variants of that matter.